website/docs/concepts/event-bridge/rule.md

```diff
@@ -3,3 +3,117 @@ sidebar_position: 2
 ---
 
 # Rule
+
+[Event Bus Rules](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-bus-to-bus.html) filter the event on a bus and then send those events to a target with optional transformation.
+
+Functionless allows typescript to be used when defining a rule.
+
+```ts
+const bus = new EventBus(stack, 'bus');
+const func = new Function<string, void>(stack, 'func', (event) => console.log(event.id));
+
+// an event bridge rule made with Functionless
+const lambdaEventsRule = bus
+  // filters the events on the bus to only event from lambda (or with a source value of `lambda`).
+  .when('lambdaEvents', event => event.source === "lambda");
+  // send all matched events to the given function.
+  .pipe(func);
+```
+
+The above is equal to the below in CDK:
+
+```ts
+declare const func: aws_lambda.IFunction;
+const bus = aws_events.EventBus(stack, "bus");
+const lambdaEventsRule = aws_events.Rule(bus, "lambdaEvents", {
+  eventBus: bus,
+  eventPattern: { source: ["lambda"] },
+});
+lambdaEventsRule.addTarget(new aws_event_targets.LambdaFunction(func));
+```
+
+:::info
+For more details on the supported schema for `Rule`s see [syntax](./syntax#event-patterns)
+:::
```

Comment:
what does :::info do?

Reply:
![image](https://user-images.githubusercontent.com/289213/171396843-b29c730f-e39b-4ae5-a872-82100cf21226.png)

Reply:
Love that! Going to use it everywhere.